services/consensus_topic_info.proto

```diff
@@ -31,60 +38,121 @@ import "duration.proto";
+     * A short description of this topic.
+     * <p>
+     * This value, if set, MUST NOT exceed 100 bytes when encoded as UTF-8.
      */
     string memo = 1;
 
-
     /**
-     * When a topic is created, its running hash is initialized to 48 bytes of binary zeros.
-     * For each submitted message, the topic's running hash is then updated to the output
-     * of a particular SHA-384 digest whose input data include the previous running hash.
-     * 
-     * See the TransactionReceipt.proto documentation for an exact description of the
-     * data included in the SHA-384 digest used for the update.
+     * The latest running hash of the topic.
+     * <p>
+     * This 48-byte field is the output of a SHA-384 digest with input
+     * data determined by the current version of the running hash algorithm
+     * used by the network.<br/>
+     * All new transactions SHALL use algorithm version `3`.<br/>
+     * The bytes of each uint64 or uint32 encoded for the hash input
+     * MUST be in Big-Endian format.
+     * <p>
+     * <hr/>
+     * If the algorithm version is '3', then the input data to the
+     * SHA-384 digest are, in order:
+     * <ol>
+     *   <li>The previous running hash of the topic (48 bytes)</li>
+     *   <li>The `topicRunningHashVersion` (8 bytes)</li>
+     *   <li>The payer account's shard (8 bytes)</li>
+     *   <li>The payer account's realm (8 bytes)</li>
+     *   <li>The payer account's number (8 bytes)</li>
+     *   <li>The topic's shard (8 bytes)</li>
+     *   <li>The topic's realm (8 bytes)</li>
+     *   <li>The topic's number (8 bytes)</li>
+     *   <li>The number of seconds since the epoch when the
+     *       `ConsensusSubmitMessage` reached consensus (8 bytes)</li>
+     *   <li>The number of nanoseconds within the second when the
+     *       `ConsensusSubmitMessage` reached consensus (4 bytes)</li>
+     *   <li>The `topicSequenceNumber` (8 bytes)</li>
+     *   <li>The output of a SHA-384 digest of the message bytes from the
+     *       `ConsensusSubmitMessage` (48 bytes)</li>
+     * </ol>
      */
     bytes runningHash = 2;
 
     /**
-     * Sequence number (starting at 1 for the first submitMessage) of messages on the topic.
+     * The current sequence number (starting at 1 for the first message)
+     * for messages on this topic.
      */
     uint64 sequenceNumber = 3;
 
     /**
-     * Effective consensus timestamp at (and after) which submitMessage calls will no longer succeed on the topic
-     * and the topic will expire and after AUTORENEW_GRACE_PERIOD be automatically deleted.
+     * The expiration time for this topic, in seconds since the epoch.
+     * <p>
+     * For this purpose, `epoch` SHALL be the UNIX epoch
+     * with 0 at `1970-01-01T00:00:00.000Z`.
      */
     Timestamp expirationTime = 4;
 
     /**
-     * Access control for update/delete of the topic. Null if there is no key.
+     * The key that MUST sign any transaction to update or modify this topic.
```

Comment:
The key that MUST sign any transaction to update or delete this topic.

Reply:
Thanks for catching that.
Will Fix.